x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/WildcardLikeTests.java

```diff
@@ -58,8 +58,9 @@ private static void addCases(List<TestCaseSupplier> suppliers) {
         for (DataType type : new DataType[] { DataTypes.KEYWORD, DataTypes.TEXT }) {
             suppliers.add(new TestCaseSupplier(" with " + type.esType(), List.of(type, type), () -> {
                 BytesRef str = new BytesRef(randomAlphaOfLength(5));
-                BytesRef pattern = new BytesRef(randomAlphaOfLength(2));
-                Boolean match = str.utf8ToString().startsWith(pattern.utf8ToString());
+                String patternString = randomAlphaOfLength(2);
+                BytesRef pattern = new BytesRef(patternString + "*");
```

Comment:
If this is for a RLIKE, shouldn't it be `.*`?
```suggestion
                BytesRef pattern = new BytesRef(patternString + ".*");
```

Reply:
It's not rlike - it's just sharing some test cases with rlike.